source/dagger/gameplay/team_game/character_controller.cpp

```diff
@@ -55,6 +60,17 @@ void Running::Run(CharacterController& ctrl_)
 		return;
 	}
 
+	// Character orientation
+	if (ctrl_.userInput.x > 0.0f)
+	{
+		ctrl_.sprite->scale.x = 3;
```

Comment:
looks like 3 is magic number here. It's better to use multiply by -1 here

Reply:
I tought about that, but if I did that, then i would have to check for the value of userInput, and then to check the current value of sprite to see whether it needs to be multiplied by -1. I'm not sure, but it feels like making code needlessly complicated.
I put this in here just to test funtionality of animations, that 3 will probably go into some variable (i. e. characterSize) in the future.